doc-Provisioning_Virtual_Machines_and_Hosts/topics/appe-creating-a-service-for-provisioning-vm.adoc

```diff
@@ -0,0 +1,169 @@
+
+[[create-service-dialog]]
+=== Creating a Service Dialog
+
+Using a service dialog you can determine which options users get to change. When users access a service, the majority of options available to them are preset and cannot be altered. A service dialog exposes some of those options to the user so that even if they are ordering a basic RHEL 7 machine, for example, they can at least choose the amount of memory, virtual CPUs, or other options available to the instance they order. In cases where certain fields must be unique, such as the name of virtual machines in Red Hat Virtualization, users must enter their own unique name for the virtual machine they choose or the operation will fail, so this field must be exposed.
+
+[NOTE]
+====
+You cannot create a service without assigning it a service dialog, so even if you do not want to expose any details to users, you must create at least a simple service dialog with a *Submit* and *Cancel* button.
+====
+
+A service dialog contains three components:
+
+* One or more *Tabs*.
+* Inside the tabs, one or more *Sections*. Note that in the previous method of creating a service dialog using the {product-title_short} user interface, *Sections* were referred to as *Boxes*.
+* Inside the sections, one or more *Elements*. Elements are controls that accept input. Elements contain methods, like check boxes, drop-down lists or text fields, to fill in the options on the provisioning dialog.
+
+[IMPORTANT]
+====
+The names of the elements must correspond to the options used in the provisioning dialog.
+====
+
+The following procedure outlines how to create a service dialog for your sample service:
+
+. Navigate to menu:Automation[Automate > Customization].
+. Click the *Service Dialogs* accordion.
+. Click image:1847.png[](*Configuration*), and then image:1862.png[](*Add a new Dialog*).
+. Enter basic information for your dialog under *General*:
+.. Enter `Provision RHV VM` in *Dialog's name*, and enter a description for the dialog in *Dialog's description*. 
+. Add a new tab to the dialog:
+.. Click image:1862.png[]*Create Tab*. Then, click the image:1851.png[pencil]icon on the new tab to edit tab information.
+.. Enter `General` in *Label*. 
+.. Optional: Enter a description for the tab in *Description*.
+.. Click *Save*.
+. Add a new section to the tab:
+.. Click image:1862.png[]*Add Section*. Then, click the image:1851.png[pencil]icon on the upper-right to edit section details.
+.. Enter `Service and VM Name` in *Label*.
+.. Optional: Enter a description for the section in *Description*.
+.. Click *Save*.
+. Add a *Text Box* element to the section for the service name:
+.. From the list of elements on the left, click the *Text Box* element, then drag-and-drop it inside the section. Then, click the image:1851.png[pencil]icon next to the element to edit its field details.
+.. Enter `Service Name` in *Label*.
+.. Enter `service_name` in *Name*
```

Comment:
Needs a period at the end of this instruction.

Reply:
Fixed.